docker/builder/Dockerfile

```diff
@@ -27,8 +27,7 @@ RUN apt-get update && \
         rustc \
         cargo \
         python3-dev \
-        python3-pip \
-        python3-virtualenv
```

Comment:
> We can no longer have virtualenv in the builder image, as it messes with the installation of our Python packages.

What about pipelines based on ODK that require the ability to install a virtual environment before running system? Seems quite fundamental a thing to remove from the image.. How does it "mess"?

Reply:
We are talking about the *builder* image here! This does not remove anything from the final ODKFull!

Having `virtualenv` installed in the builder image creates a mess because

> the Ubuntu package for virtualenv automatically installs platformdirs version 2.5.1, which then prevents us from installing the platformdirs 4.x that we need as a dependency for some of our packages.